CRM/Contact/Import/ImportJob.php

```diff
@@ -411,7 +411,7 @@ public static function getIncompleteImportTables() {
     $database = $dao->database();
```

Comment:
If you trace back this function I think it can be removed entirely + the one function & one class that calls it